src/test/kotlin/no/nav/klage/clients/KlageKafkaConsumerTest.kt

```diff
@@ -0,0 +1,217 @@
+        val vedtak: String,
+        val saksnummer: String?,
+        val dato: LocalDate,
+        val begrunnelse: String,
+        val tema: String,
+        val ytelse: String,
+        val vedlegg: List<Vedlegg> = emptyList(),
+        var fileContentAsBytes: ByteArray? = null
+    )
+
+//    data class KlageAnkeInput(
+//        val id: Int,
+//        val identifikasjonstype: String,
+//        val identifikasjonsnummer: String,
+//        val klageInstans: Boolean?,
+//        val trygderetten: Boolean?,
+//        val fornavn: String,
+//        val mellomnavn: String,
+//        val etternavn: String,
+//        val adresse: String,
+//        val telefon: String,
+//        val vedtak: String,
+//        val dato: LocalDate,
+//        val begrunnelse: String,
+//        val tema: String,
+//        val ytelse: String,
+//        val vedlegg: List<Vedlegg> = emptyList(),
+//        var fileContentAsBytes: ByteArray? = null,
+//        //deprecated, only used to parse old kafka entries
+//        val saksnummer: String? = null,
+//        val userChoices: List<String>? = emptyList(),
+//        val userSaksnummer: String?,
+//        val internalSaksnummer: String?,
+//        val fullmektigNavn: String?,
+//        val fullmektigFnr: String?,
+//        val klageAnkeType: KlageAnkeType = no.nav.klage.domain.KlageAnkeType.KLAGE,
+//        val previousUtfall: String?
+//    )
+
+    val oldKlageInput = OldKlage(
+        id = 123,
+        identifikasjonstype = "FNR",
+        identifikasjonsnummer = "12345678901",
+        klageInstans = false,
+        trygderetten = false,
+        fornavn = "Navn",
+        mellomnavn = "",
+        etternavn = "Navnesen",
+        adresse = "Gateveien 123",
+        telefon = "98765432",
+        vedtak = "",
+        dato = LocalDate.now(),
+        begrunnelse = "BEGRUNNELSE",
+        tema = "OMS",
+        ytelse = "OMS",
+        saksnummer = "123"
+    )
+
+    val input = KlageAnkeInput(
+        id = 123,
+        identifikasjonstype = "FNR",
+        identifikasjonsnummer = "12345678901",
+        klageInstans = null,
+        trygderetten = null,
+        fornavn = "Navn",
+        mellomnavn = "",
+        etternavn = "Navnesen",
+        adresse = "Gateveien 123",
+        telefon = "98765432",
+        vedtak = "",
+        dato = LocalDate.now(),
+        begrunnelse = "BEGRUNNELSE",
+        tema = "OMS",
+        ytelse = "OMS",
+        userSaksnummer = null,
+        internalSaksnummer = null,
+        fullmektigNavn = null,
+        fullmektigFnr = null,
+        previousUtfall = null
+    )
+
+    val mapper = ObjectMapper()
+        .registerModule(JavaTimeModule())
+        .registerModule(KotlinModule())
+
+    @Test
+    fun `Innsending med utdatert modell blir deserialisert`() {
+        val inputString = mapper
+            .writeValueAsString(oldKlageInput)
+        val parsedKlage = inputString.toKlage()
+        assertTrue(parsedKlage.id == oldKlageInput.id)
+    }
+
```

Comment:
Burde det vært en test for å sjekke om input er deprecated? Kanskje den under kan brukes til det.